### PR TITLE
[UI] [FAVICON] Adds profiles to head tag to define favicon

### PR DESCRIPTION
**Pull request by ajainarayanan**

### CODE REVIEW

File: cdap-ui/app/cdap/cdap.html [hunk 1]
```diff
@@ -16,9 +16,12 @@
 
 <!DOCTYPE html>
 <html lang="en">
-<head>
+<head profile="http://www.w3.org/2005/10/profile">
     <title></title>
 
+    <link rel="icon"
+      type="image/png"
+      href="/cdap_assets/img/favicon.png">
```
Comment (elfenheart) [inline cdap-ui/app/cdap/cdap.html]: this is forcing favicon to be from this path. We have defined a favicon path in theme. Can we make this dynamic?

**Comment by itsanudeep** [comment]
@ajainarayanan profile attribute is not supported in HTML5. Is this not going to be a problem?

File: cdap-ui/app/cdap/cdap.html (hunk 1 above)
Comment (ajainarayanan) [reply cdap-ui/app/cdap/cdap.html]: While configuring respective themes we need to make sure the favicon file name remains the same. Until now we had different names for favicon and trying to accommodate that in the code however a favicon is a very unique type of image being used by the browser. Doesn't make much sense to have different themes have different names for it. We can enforce this and have the environment setting up the theme to overwrite the existing file.

**Comment by ajainarayanan** [comment]
@itsanudeep where are you seeing this?

**Comment by itsanudeep** [comment]
> @itsanudeep where are you seeing this?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head

File: cdap-ui/app/cdap/cdap.html (hunk 1 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/cdap.html]: then the favicon configuration should be removed from the theme file. The express server is also serving a favicon that should be removed

**Review by elfenheart — APPROVED** [review]
👍 LGTM

File: cdap-ui/app/cdap/cdap.html (hunk 1 above)
Comment (ajainarayanan) [reply cdap-ui/app/cdap/cdap.html]: Filed [CDAP-16099](https://issues.cask.co/browse/CDAP-16099) JIRA for cleanup to be done later.

**Pull request merged**